remoteworkitem/remoteworkitem.go

```diff
@@ -11,7 +11,7 @@ const (
 	SystemRemoteItemID = "system.remote_item_id"
 	SystemTitle        = "system.title"
 	SystemDescription  = "system.description"
-	SystemStatus       = "system.status"
+	SystemStatus       = "system.state"
```

Comment:
@baijum 
This was changed because it wasn't inconsistent with those with our seeded work item types. OK - will change the const name as well.